api/users/views.py

```diff
@@ -662,3 +662,20 @@ def perform_destroy(self, instance):
             if val['id'] in current_institutions:
                 user.remove_institution(val['id'])
         user.save()
+
+class UserPublicFiles(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin):
+    view_name = 'user-public-files'
+    view_category = 'users'
+    serializer_class = PublicFilesSerializer
+
+    required_read_scopes = [CoreScopes.USERS_READ, CoreScopes.NODE_REGISTRATIONS_READ]
```

Comment:
Why should reading a node containing a users public files require `NODE_REGISTRATIONS_READ`?